### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by Ridermansb**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Ridermansb/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
```
Comment (Ridermansb) [inline ReadMe.md:3]: This maybe can put in footer

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+To add a flag simply leave a comment with your desired flag. For
+convenience we also have a dropdown menu to add these comments
+automatically.
+
+Another key feature you need to know about is ninja stars. Ninja 
+stars are how you communicate that a pull request is okay to merge. 
+You can set, by repository, how many ninja stars you think pull 
+requests should have before merging.
+
+**To add a ninja star**, click the **ninja star** button.
```
Comment (Ridermansb) [inline ReadMe.md:31]: **Add a ninja star**, clicking  on ninja star*\* button.

File: ReadMe.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
```
Comment (Ridermansb) [inline ReadMe.md:15]: !fix try

Comment (Ridermansb) [inline ReadMe.md:15]: Now is !fixed

**Pull request merged**